packages/gguf/src/types.ts

```diff
@@ -138,4 +138,5 @@ export interface GGUFTensorInfo {
 export interface GGUFParseOutput {
 	metadata: GGUFMetadata;
 	tensorInfos: GGUFTensorInfo[];
+	paramCount?: number;
```

Comment:
why not the same prop name as for safetensors (`parameterCount`)? Besides i don't think this should be in this type (which is strictly what is in GGUF), but just in the function return type